test/extended/cluster/cl.go

```diff
@@ -122,11 +126,32 @@ var _ = g.Describe("[Feature:Performance][Serial][Slow] Load cluster", func() {
 					}
 					// TODO sjug: pass label via config
 					labels := map[string]string{"purpose": "test"}
-					CreatePods(c, pod.Basename, nsName, labels, config.Spec, pod.Number, tuning)
+					err := CreatePods(c, pod.Basename, nsName, labels, config.Spec, pod.Number, tuning, &pod.Sync)
+					o.Expect(err).NotTo(o.HaveOccurred())
 				}
 			}
 		}
 
+		if sync.Running {
+			for _, ns := range namespaces {
+				err := SyncRunningPods(c, ns, sync.Selectors, time.Duration(sync.Timeout)*time.Second)
```

Comment:
```
time.Duration(sync.Timeout)*time.Second
```

This kind of pattern has happened a lot in here, I suggest using `time.ParseDuration` rather than hardcoding. This is something I've done in the [perf-tests cluster loader version](https://github.com/kubernetes/perf-tests/tree/master/clusterloader).